roofit/roofitcore/src/RooAbsArg.cxx

```diff
@@ -2275,6 +2275,27 @@ Bool_t RooAbsArg::addOwnedComponents(const RooArgSet& comps)
 }
 
 
+////////////////////////////////////////////////////////////////////////////////
+/// Take ownership of the contents of 'comps'. Different from the overload that
+/// taked the RooArgSet by `const&`, this version can also take an owning
+/// RooArgSet without error, because the ownership will not be ambiguous afterwards.
+
+bool RooAbsArg::addOwnedComponents(RooAbsCollection&& comps)
+{
+  if (!_ownedComponents) {
+    _ownedComponents = new RooArgSet("owned components") ;
+  }
+  return _ownedComponents->addOwned(std::move(comps)) ;
+}
+
+
+bool RooAbsArg::addOwnedComponents(const RooArgList& comps) {
```

Comment:
Add doxygen.

Reply:
You can use `\copydoc` if you just want to copy the docs from above.

Reply:
Right, I have added the doxygen now. Sorry for not adding it before, I first wanted to see the CI tests pass before writing the docs